packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -35,6 +35,14 @@ export const NotificationOptionItem: OptionItem = {
   detail: getLocalizedString("core.NotificationOption.detail"),
 };
 
+export const CommandAndResponseOptionItem: OptionItem = {
+  id: "CommandAndResponse",
+  label: "Command and Response",
```

Comment:
Shall we call it `Command and Response Bot` here to let users know they're going to build a Teams Bot?

Reply:
Sure, updated.